### DISCUSSION TITLE
Updated to version 12 and custom keymaps were overwritten

### DISCUSSION THREAD

**Discussion by jacksonmelcher** [LazyVim/LazyVim#4140] [Q&A] [answered]
Hi, I just updated to v 12 and one of my keymaps was overwritten.  I had "save file" mapped to `<leader>w`  but now it is mapped to window. 

I added this but it doesn't work and it still shows window is mapped to `<leader>w` in which key view when I press my <leader> key.

```
-- Remove default <leader>w mapping
vim.keymap.del("n", "<leader>w")

-- Add custom mapping to save file
vim.keymap.set("n", "<leader>w", ":w<CR>", { desc = "Save file" })
```
Which-key commit `cc54b75` 
LazyVim: `12.36`

**Comment by folke** [comment, marked as answer]
for which-key, add `opts.spec = {{"<leader>w", desc = "save file"}}`

**Reply by jacksonmelcher** [reply]
This should go in `plugins/which-key.lua`?

**Reply by dotfrag** [reply]
Yes

**Reply by jacksonmelcher** [reply]
Thank you
